docs/trackers/core/reid/reid.md

```diff
@@ -0,0 +1,43 @@
+# ReID Model
```

Comment:
That header is no longer valid. We talk about a lot more than just `ReID Model`. We need to make it more general.

Reply:
Updated the header.